pkg/services/featuremgmt/registry.go

```diff
@@ -157,5 +157,11 @@ var (
 			Description: "Lock database during migrations",
 			State:       FeatureStateBeta,
 		},
+		{
+			Name:            "dbFileStorage",
```

Comment:
should this just be "filestore"? the db part feels independent (and should likely come along for the ride once ready)

Reply:
Is there a need for a generic `filestorage` feature flag? It's just a service that can be used in any scenario - I feel like  feature-specific flags such as `storeDashboardsInFs` would be more appropriate if we want to gate the access

I added `dbFileStorage` flag to signal that 1) it can't be yet used as the default storage backend and 2) its internal details (table structure) are still evolving/in alpha

Reply:
ahh -- i have not dug into how the flag is used :)

I think we want a general flag for the whole system "filestorage" that will gate disallow access to any apis etc while we sort our development.  I think we can later remove that when we are confident that "../../" is safe etc.

I don't know if we need a specific `dbFileStorage` since I think we should only merge the SQL migration (enabled) once we are willing to have it be a standard part of the storage solution anyway.   And then yes -- we will need a flag or some config for `storeDashboardsInFS` or whatever plans we have :)